### ISSUE TITLE
Voice Messages?

### ISSUE THREAD

**Issue by IngwiePhoenix** [mautrix/whatsapp#332]
Hello!

I have been using this bridge for nearly a whole year now. Matrix has become my go-to for messaging - and the only thing missing is Discord puppeting. Well, that, and sending voice messages.

So, to make a long story short: I am nearly blind and can't work my phone very well, especially typing is a pure chore. So I would like to know if it would be possible to send voice messages? Element doesn't have support for this (yet?) but alternatives like FluffyChat do.

Would it be possible to send voice messages using this bridge? When I do so from Fluffy, they end up being shown as downloadable media files instead of voice messages.

Kind regards,
Ingwie

**Comment by thallian** [comment]
Element-Android does have support for sending voice messages since v1.1.15. It's quite new, so not yet on f-droid, if you are getting it from there, might be on the playstore already though (I built it myself). 

I verified just now, that voice messages sent from element-android to whatsapp over the bridge do indeed turn up as if you had recorded them in whatsapp (the reverse is not yet as convenient, though in the web version of element it already is). Hope that helps a bit :)
